src/loader/bgw_launcher.c

```diff
@@ -397,49 +421,84 @@ stopped_db_schedulers_cleanup(HTAB *db_htab)
 	}
 }
 
-/* Actions for message types we could receive off of the bgw_message_queue*/
-static void
-message_start_action(HTAB *db_htab, BgwMessage *message, VirtualTransactionId vxid)
+static DbHashEntry *
```

Comment:
doc: finds the hashtable entry for a database's scheduler, allocating one if it does not exist.

Reply:
done